MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/Utilities/MarsDate.cs

```diff
@@ -65,10 +65,35 @@ public static double MarsToEarthDate(DateTime marsDate)
             return earthSolDate;
 
         }
-        public static double RoverSol(DateTime roverDate)
+        //the time the different rovers have been on mars 
+        public static double CuriositySol(DateTime roverDate)
```

Comment:
Nice. Coming along well.

I still think that these should probably be integers though.
And perhaps we can make it read even more cleanly if we separate out some constants for the landing dates?

```csharp
DateTime perserveranceLandingDate = new DateTime(2021, 02, 18);

public static int PerseveranceSol(DateTime date)
{
    var daysSinceLanding = date.Subtract(perserveranceLandingDate).TotalDays;
    return (int)(daysSinceLanding * earchConversionRate);
}
```

Reply:
it;s not happy now, getting this  " can not declare instance members in a static class"

Reply:
oh yeah, everything is static so you'll need to make the constant static too. 

```csharp
private static DateTime perserveranceLandingDate = new DateTime(2021, 02, 18);
```

Reply:
any reason to have it private and not static?